spec/features/fill_in_user_form_spec.rb

```diff
@@ -70,6 +70,18 @@
     expect(page.find('#user_date_of_birth_3i').value).to eq('2')
   end
 
+  it 'finds and fills a spanish locale date field' do
+    I18n.locale = :es
+    visit 'user_form'
+    form = Formulaic::Form.new(:user, :new, date_of_birth: Date.new(1980, 1, 2))
+    form.fill
+
+    expect(page.find('#user_date_of_birth_1i').value).to eq('1980')
+    expect(page.find('#user_date_of_birth_2i').value).to eq('1')
+    expect(page.find('#user_date_of_birth_3i').value).to eq('2')
+    I18n.locale = :en
```

Comment:
Since `expect` will raise an error and the RSpec runner will catch it, we need to ensure this global state is reset. My suggestion:

```
it '...' do
  begin
    I18n.locale = :es
    # the rest of the spec
  ensure
    I18n.locale = :en
  end
end
```